### ISSUE TITLE
Changing theme

### ISSUE THREAD

**Issue by convergenceIM** [convergenceIM/alpha-scientist#15]
I've spent far too much time today looking at the different pelican themes.  Huge variety, none is perfect.  How complex would it be to use octopress but with two modifications:
1) change the code block coloring to look like cells in `tuxlite_tbs` or `nikhil-theme`?
2) possible to make the code blocks wrap dynamically like in `tuxlite-tbs`?

**Comment by yn-coder** [comment]
I think we could combine it. One possible restriction - it would be our own fork of octopress, and upgrading for our fork could take some time each time when octopress will updates.

**Comment by convergenceIM** [comment]
OK, let;s try it then.  I like everything about Octopress theme except for those code blocks.  I really don't think I'd need/want to change it much if at all

**Comment by yn-coder** [comment]
octopress

![octopress ml for stock market prediction part 1 data management the alpha scientist -_2018-06-30_23-42-22](https://user-images.githubusercontent.com/8451944/42128959-bce5dc64-7cbf-11e8-8283-a6260a855275.png)

nikhil

![nikhil ml for stock market prediction part 1 data management - the alpha scientist -_2018-06-30_23-41-04](https://user-images.githubusercontent.com/8451944/42128960-c209d222-7cbf-11e8-862c-0c7c9d70d1ff.png)

tuxlite_tbs

![tuxlite_tbs the alpha scientist - ml for stock market prediction part 1 data management -_2018-06-30_23-44-41](https://user-images.githubusercontent.com/8451944/42128961-c5a34fd0-7cbf-11e8-9bf1-ed1288a2c433.png)

**Comment by convergenceIM** [comment]
Yes.  I'd like to use octopress _except_ with making the jupyter notebook code cells look like tuxlite.

**Comment by yn-coder** [comment]
notes for me...

to change the octopress theme we need to 

1. Install SASS
2, Install Ruby
3. Install Compass
4, Change and recompile the octopress theme
5. debug

**Comment by convergenceIM** [comment]
That looks great.  Doesn't need to be a perfect match to tuxlite, just not that darkula color scheme, which I find really hard to read

**Comment by yn-coder** [comment]
@convergenceIM Do publish it to master?

**Comment by convergenceIM** [comment]
Yes, I publish to master

**Comment by yn-coder** [comment]
Ive made the changes local. I can publish changed octopress theme and new site

**Comment by yn-coder** [comment]
Pablished, Please review

**Comment by convergenceIM** [comment]
The style looks GREAT.  No issues with font, color, alignment, etc...  Much better than the stock theme, IMO.  A few issues with other parts of the site:
1. Disqus comments aren't appearing
2. Social links aren't appearing (does the pelicanconf/publishconf file you're using contain the social links?)
3. Need to add an "About" tab next to the Blog tab on the menu bar.  I may also want to add another non-blog page ("Consulting").  
4. Also want to embed the twitter feed in the right sidebar (the [docs](https://github.com/duilio/pelican-octopress-theme/blob/master/README.rst) make it look like they've already built this functionality into the theme
5. Add the sign-up form
6. Looks like the [docs](https://github.com/duilio/pelican-octopress-theme/blob/master/README.rst) also support several features that I'd like to make use of:
TWITTER_USER: data2alpha
FACEBOOK_LIKE: True
7. Try adding the png image we used with the Alchemy theme in the right sidebar.  Not sure how it'll look.

**Comment by convergenceIM** [comment]
On second thought, the png image (7) may be too cluttered.  If it can be added to the top banner, then maybe worth it, but probably not the sidebar.  The sidebar should already have: 
- Recent Posts
- Categories/Tags
- social
- twitter feed for @data2alpha
- mailing list signup form

**Comment by convergenceIM** [comment]
Actually, one question about format (as in, "is this a small or big thing to do" kind of question):

Is the color palate of the theme easily changed with a line or two in the css?  If easy, I generally like dark blues (like #203c63) and would be interested in seeing how it looked with the dark blue for banner (instead of dark grey) and with the same dark blue as the default font color.  

If this involves rebuilding from Ruby, then that may be more complexity than I'd want to add to this process

**Comment by convergenceIM** [comment]
One other nit: possible to change the font size of the titles for blog posts (they're generated by pelican so think it needs to be within the theme itself)?  They're much too big right now and would want to make them smaller by about 1/3.  

Related, I still don't understand whether changing fonts for headers, body etc... is a simple change within a CSS sheet or if it needs to be done within Ruby etc...  I'd like to be able to play with settings if simple HTML/CSS but much less motivatd to do the latter

**Comment by yn-coder** [comment]
It should be easy with sass recompiling. I'll try to rebuild the css

**Comment by convergenceIM** [comment]
OK, sounds good.  Also see  #27 for an issue with the blog postings losing their black on both sides framing.
